packages.yml

```diff
@@ -3,7 +3,7 @@ packages:
 #   version: [">=0.7.0", "<0.8.0"]
 
   - git: https://github.com/fivetran/dbt_shopify_source.git
-    revision: feature/package-revamp
+    revision: feature/revamp/revert-timestamp-name-change
```

Comment:
https://github.com/fivetran/dbt_shopify_source/pull/42

Reply:
The usual "make sure to change the revision" when merging =)

Reply:
The usual "make sure to change the revision" when merging =)